masterlist.yaml

```diff
@@ -6693,6 +6699,9 @@ plugins:
       # version: 2.1
       - crc: 0xBBA1C2CA
         util: 'SSEEdit v4.0.2'
+  - name: 'EconomyOverhaul CV - (Vokrii|Ordinator) Patch.esp'
+    after:
+      - 'EconomyOverhaul CV - MorrowLootUltimate Patch.esp'
```

Comment:
Group order should cover this after, but there's no harm in having it since it's correct.